index.js

```diff
@@ -132,6 +136,7 @@ ConfirmHcaptcha.defaultProps = {
   size: 'invisible',
   passiveSiteKey: false,
   showLoading: false,
+  closableLoading: false,
```

Comment:
```suggestion
  closableLoading: false,
```

why default to false?

Reply:
To not make it "breaking" change

In general, there are no objections to making it `true`, but so far we got only one feedback #65 where the user asked to make it closable

Reply:
I'd prefer a reasonable default. It should self-close after a large N (10s?) if we don't make it closable by default.

Reply:
It makes sense, will add this logic

Reply:
After testing I decided to increase default timeout to 15sec